packages/react/src/components/iconCard/tests/IconCard.spec.tsx

```diff
@@ -267,4 +270,27 @@ describe('IconCard', () => {
         const card = container.querySelector('.icon-card');
         expect(card).not.toHaveClass('animateOnHover');
     });
+
+    it('should NOT render the badges within the main content container if placeBadgesAbove is unset', () => {

```

Comment:
```suggestion
    it('does not render the badges within the main content container if placeBadgesAbove is unset', () => {

```

Reply:
I get what you mean by unset, but I think false is more clear :see_no_evil:

Reply:
Yep totally fair